test/tools/unified-spec-runner/match.ts

```diff
@@ -131,7 +131,10 @@ TYPE_MAP.set(
 );
 TYPE_MAP.set(
   'long',
-  actual => (typeof actual === 'number' && Number.isInteger(actual)) || Long.isLong(actual)
```

Comment:
Out of curiosity, is this to fix any failing tests or is this an improvement for the future? I don't see any unified tests being run with bigint enabled, so I'm surprised this is necessary

Reply:
If I remember correctly, this was done to fix tests that were failing when `useBigInt64` was globally defaulted to true. @nbbeeken correct me if I'm wrong, but I believe it was to fix a failure in one of the client-side encryption tests.

Reply:
Correct, this was done when we were setting `useBigInt64` to true globally, this just helped cut out a lot of expected failures so we could better sort through the actual issues

Reply:
OK. To be clear here, I'm in favor of this change because it's needed if we ever want to enable bigint serialization in our unified runner.

But I want to confirm that we are _not_ setting this option globally for our tests, correct?

Reply:
Just to document it, we are _not_ setting the option to `true` globally, but we ran evergreen patches where that was the case just to document the test failures that occurred and make changes to the code/tests that made those failures more transparent, so rather than just resulting in an obscure error, it would result in something more readable, like this example: https://parsley.mongodb.com/test/mongo_node_driver_next_rhel80_large_hydrogen_test_3.6_replica_set_patch_e544d880b1731fa59ad6a0d740b946e79478cfa6_63efd6123627e0388b69d98e_23_02_17_19_31_41/0/41766c6f7a9248879f6067a39be845a5?bookmarks=0,8&shareLine=1